### PR TITLE
translate concepts part into chinese

### PR DESCRIPTION
**Pull request by Johnny-Three**
Signed-off-by: JohnnyThree <where@gmail.com>

### CODE REVIEW

File: content/zh/docs/content/cell.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Cell
+description: 数据中心, 可用区域或计算资源组
+---
+一个Cell是指一个网络资源区域，在这个区域里，放置一组服务器或网络基础设施。它通常对应物理部署中的数据中心，有时称为*区域*或者*可用区域*。Cell的主要作用是网络隔离。Vitess可以优雅的处置Cell级别的网络故障，如果有其他的Cell故障并不会影响到当前Cell。
+
+
+Vitess中的每个Cell里都有一个[本地拓扑服务托管服务集群](#topology-service)，本地拓扑服务负责存储和管理注册在此Cell下的所有网络资源的相关元数据信息。其中包括这个Cell下所有的路由及vttabelt的信息。这样的设计使得Cell更加易于模块化管理，方便拆卸和组装。
```
Comment (morgo) [inline content/zh/docs/content/cell.md]:
```suggestion
Vitess中的每个Cell里都有一个[本地拓扑服务托管服务集群](../topology-service)，本地拓扑服务负责存储和管理注册在此Cell下的所有网络资源的相关元数据信息。其中包括这个Cell下所有的路由及vttablet的信息。这样的设计使得Cell更加易于模块化管理，方便拆卸和组装。
```
There is a broken link - I have a PR currently open to fix english docs. Also s/vttabelt/vttablet/

File: content/zh/docs/content/keyspace-id.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Keyspace ID
+---
+*keyspace ID*是用于确定给定行所在的分片的值。 [基于范围的分片]（../../分片/＃键范围和分区）是指创建分片，每个分片覆盖特定范围的keyspace ID。
```
Comment (morgo) [inline content/zh/docs/content/keyspace-id.md]: I don't believe the file name here should be translated.

Comment (Johnny-Three) [reply content/zh/docs/content/keyspace-id.md]: ah. will correct after you review it all.

Comment (morgo) [reply content/zh/docs/content/keyspace-id.md]: Rest lgtm

Comment (Johnny-Three) [reply content/zh/docs/content/keyspace-id.md]: understood.

**Comment by Johnny-Three** [comment]
@morgo   PTAL

**Pull request merged**